### ISSUE TITLE
Dependency error when creating a new project

### ISSUE THREAD

**Issue by JonathanPiquard** [NativeScript/nativescript-cli#5482]
Hi ! **I'm getting an error when I'm trying to create a new Nativescript project.**

Nativescript : 7.1.2
Angular CLI: 11.1.0
Node: 15.5.1
OS: linux x64

![Dependency error](https://user-images.githubusercontent.com/26177563/105510065-80de4200-5cce-11eb-9a9a-3f31b72b40c0.png)

**I attempted to resolve the issue this way :**
I've cloned the template from github and then changed the version of the angular dependencies to 11.1.0 as suggested in another issue (https://github.com/angular/angular/issues/39094) but then when I tried to run 'npm update' I've got another dependency issue with typescript this time.

**Comment by rigor789** [comment]
See https://github.com/NativeScript/NativeScript/issues/9152#issuecomment-762445495

**tl;dr**
use `node 14` & `npm 6` until support is implemented for 15 & 7

**Comment by JonathanPiquard** [comment]
It works ! Thank you :D

**Comment by JonathanPiquard** [comment]
I've gotten straight into this issue. It's been months it's unresolved. Will you fix it ? There are no answer from the telerik team.
https://github.com/NativeScript/nativescript-schematics/issues/290

**Comment by olaracuente** [comment]
Any update on this issue. 
I have an apple M1. I was able to created a working project which is working(IOS,Android, Web), but suddenly when I tried to create a new one I get all the errors mentioned in this issue.

NativeScript version 7.2.0
Angular CLI: 11.2.0
npm version 7.5.2
node version 15.8.0

**Comment by rigor789** [comment]
> Any update on this issue. 
> I have an apple M1. I was able to created a working project which is working(IOS,Android, Web), but suddenly when I tried to create a new one I get all the errors mentioned in this issue.
> 
> NativeScript version 7.2.0
> Angular CLI: 11.2.0
> npm version 7.5.2
> node version 15.8.0

Use npm 6 as mentioned throughout the issue, and the pinned/linked issue!
